assembly/runtime/collections/persistentDeque.ts

```diff
@@ -71,21 +100,51 @@ export class PersistentDeque<T> {
   }
 
   /**
-  * @param index The index to check.
-  * @returns True if the given index is within the range of the deque indices.
-  */
+   * Checks whether the index is within the range of the deque indices
+   *
+   * ```ts
+   * let dq = PersistentDeque<string>("q")
+   *
+   * dq.containsIndex(0) // false
+   * dq.pushFront("hello world")
+   * dq.containsIndex(0) // true
+   * ```
+   *
+   * @param index The index to check.
+   * @returns True if the given index is within the range of the deque indices.
+   */
   containsIndex(index: i32): bool {
     return index >= 0 && index < this.length;
   }
 
   /**
-  * @returns The length of the deque.
-  */
+   * Returns the length of the deque
+   *
+   * ```ts
+   * let dq = PersistentDeque<string>("q")
+   *
+   * dq.length // 0
+   * dq.pushFront("hello world")
+   * dq.length // 1
+   * ```
+   *
+   * @returns The length of the deque.
```

Comment:
Seems redundant with first line

Reply:
yes, lots of this kind of thing here.  what would be better?